extensions/blocks/dialogue/save.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * External dependencies
+ */
+
+export default function save() {
+	return null;
```

Comment:
Did you create this file in preparation of a saved content? If you're not planning on anything here for now, maybe we could keep things simple and save nothing in `extensions/blocks/dialogue/index.js`?

Reply:
makes sense to me. 👍